ai/orchestrators/models-config.mdx

```diff
@@ -37,6 +45,14 @@ currently **recommended** models and their respective prices.
     "model_id": "openai/whisper-large-v3",
     "price_per_unit": 12882811
   },
+  {
+    "pipeline": "segment-anything-2",
+    "model_id": "facebook/sam2-hiera-large",
+    "price_per_unit": 3.565,
```

Comment:
Looks like we need to change this to an integer value, so something like :

```
    "price_per_unit": 3565,
    "pixels_per_unit": 1e-13,
```

Reply:
Good catch!